pkg/controllers/user/logging/utils/kafka.go

```diff
@@ -239,6 +249,18 @@ func (w *kafkaTestWrap) kafkaConn(dial dialer.Dialer, config *tls.Config, smartH
 	return kafka.NewConn(conn, w.Topic, defaultPartition), nil
 }
 
+func (w *kafkaTestWrap) verifyKafkaCert() error {
+	isSelfSigned, err := IsSelfSigned([]byte(w.Certificate))
+	if err != nil {
+		return err
+	}
+
+	if IsClientAuthEnaled(w.ClientCert, w.ClientKey) && isSelfSigned {
```

Comment:
nit: putting isSelfSigned as a first check in if can help to save on calling isClientAuthEnabled when isSelfSigned is false.